dev-util/geany-plugins/geany-plugins-1.34.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 vala
+
+DESCRIPTION="A collection of different plugins for Geany"
+HOMEPAGE="https://plugins.geany.org"
+SRC_URI="https://plugins.geany.org/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="+gtk3 ctags debugger enchant git gpg gtkspell lua markdown multiterm nls pretty-printer python scope soup"
+REQUIRED_USE="
+	gtk3? ( !debugger !multiterm !python )
+	!gtk3? ( !markdown )
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+COMMON_DEPEND="
+	>=dev-util/geany-1.34[gtk3=]
```

Comment:
Could you please sort this alphabetically? :) 

Unconditional before conditionals, as you've done, but both sorted alphabetically please.

Reply:
Swapped glib and geany.

gtk3 is a special case because all the other conditionals are plugins.